modules/userId/index.js

```diff
@@ -490,6 +490,9 @@ function mkPriorityMaps() {
         for (const {allowed, bidders, module} of modules) {
           const value = module.idObj?.[key];
           if (value != null) {
+            if (!dep.isAllowed(ACTIVITY_ENRICH_EIDS, activityParams(MODULE_TYPE_UID, module?.config?.name, {init: false}))) {
+              return null;
```

Comment:
I believe this is even more aggressive than before - if we run into a disallowed module, even one that would not normally be used for this ID,  we don't look into other modules. `continue` would ignore this module but let other lower priority modules provide the ID, which is what would happen when you disable the module entirely.